src/_colors.scss

```diff
@@ -1,27 +1,25 @@
-:root {
-  --color-heading: #3f434a;
-  --color-shadow: #a4a5a566;
-  --color-body-light: #8a9099;
-  --color-body: #595f69;
-  --color-grey-light: #eeeeee;
-  --color-grey: #c6c9cc;
-  --color-border: #e8e9eb;
-  --color-bg: #f8f8f8;
-  --color-bg-light: #efeff1;
-  --color-primary: #304ffd;
-  --color-primary-hover: #445ffc;
-  --color-primary-light: rgba(48, 79, 253, 0.1);
-  --color-primary-dark: #263fca;
-  --color-danger: #ff5950;
-  --color-danger-hover: #fe6f67;
-  --color-danger-light: rgba(253, 121, 114, 0.1);
-  --color-danger-dark: #ea2b21;
-  --color-warning: #ff965d;
-  --color-warning-hover: #ffa778;
-  --color-warning-light: rgba(229, 110, 25, 0.1);
-  --color-warning-dark: #e56e19;
-  --color-success: #49c96d;
-  --color-success-hover: #79cc91;
-  --color-success-light: rgba(73, 201, 109, 0.1);
-  --color-success-dark: #27b950;
-}
+$color-heading: #3f434a;
+$color-shadow: #a4a5a566;
+$color-body-light: #8a9099;
+$color-body: #595f69;
+$color-grey-light: #eeeeee;
+$color-grey: #c6c9cc;
+$color-border: #e8e9eb;
+$color-bg: #f8f8f8;
+$color-bg-light: #efeff1;
+$color-primary: #304ffd;
+$color-primary-hover: #445ffc;
+$color-primary-light: rgba(48, 79, 253, 0.1);
+$color-primary-dark: #263fca;
+$color-danger: #ff5950;
+$color-danger-hover: #fe6f67;
+$color-danger-light: rgba(253, 121, 114, 0.1);
+$color-danger-dark: #ea2b21;
+$color-warning: #ff965d;
+$color-warning-hover: #ffa778;
+$color-warning-light: rgba(229, 110, 25, 0.1);
+$color-warning-dark: #e56e19;
+$color-success: #49c96d;
+$color-success-hover: #79cc91;
+$color-success-light: rgba(73, 201, 109, 0.1);
+$color-success-dark: #27b950;
```

Comment:
In combination with **_correct_** use of `@use "..." as ...` we no longer need to prefix our variables with their type – `color-` in this case.